src/core/languages.js

```diff
@@ -621,3 +621,5 @@ export default config.get('langs').reduce((object, locale) => {
 
   return object;
 }, {});
+
+export const languagesUnfiltered = languages;
```

Comment:
I dont think we should be doing this.

Reply:
Can you expand on why?

Reply:
I think we should keep the default export. You can make the distinction between filtered (or is sorted?) and not filtered/sorted only where you need this distinction.

It should result in less changes too.

Reply:
The default export is completely different data.

Reply:
@willdurand see https://github.com/mozilla/addons-frontend/blob/92bd46069a2c9fa4bd0d804f496d17c894657e73/src/core/languages.js#L617

Reply:
oh sorry for that, I did not pay attention to that. What does this code do actually?

any chance to come up with a meaningful name and assign it as default? That would make things easier to understand, and we could `export` the `languages` array (which could be renamed too)

```
export const allLanguages = {};

const reducedLanguages = ...;

export default reducedLanguages;
```

Reply:
This is why the original code doesn't work, because it's filtering this list of languages to match the supported locales for the site. This is mentioned in the description. The reason for a distinct export rather than exporting the raw data is to prevent the case of someone making assumptions about it.

Reply:
I think this code is mainly used by the locale switcher.

Reply:
I see, but maybe you could directly rename `languages` to `unfilteredLanguages` and export it in `src/core/languages.js`?

Reply:
Yep, that would work - I'll see about doing that in a follow-up patch.